src/app/sb-admin/components/pages/sb-user/sbuser.component.html

```diff
@@ -5,24 +5,28 @@
             <p-toolbar styleClass="mb-4">
                 <ng-template pTemplate="left">
                     <div class="my-2">
-                        <button pButton pRipple label="Add User" icon="pi pi-plus" class="p-button-success mr-2" (click)="openNew()"></button>
-                        <button pButton pRipple label="Delete" icon="pi pi-trash" class="p-button-danger" (click)="deleteSelectedUsers()" [disabled]="!selectedUsers || !selectedUsers.length"></button>
+                        <button pButton pRipple label="{{ 'ADD_USER' | i18next }}" icon="pi pi-plus" class="p-button-success mr-2"
+                            ></button>
+                        <button pButton pRipple label="{{ 'DELETE_USER' |i18next }}" icon="pi pi-trash" class="p-button-danger"
+                            [disabled]="!selectedUsers || !selectedUsers.length"></button>
                     </div>
                 </ng-template>
-
-                <ng-template pTemplate="right">
-                    <p-fileUpload mode="basic" accept="image/*" [maxFileSize]="1000000" label="Import" chooseLabel="Import" class="mr-2 inline-block"></p-fileUpload>
-                    <button pButton pRipple label="Export" icon="pi pi-upload" class="p-button-help" (click)="dt.exportCSV()"></button>
-                </ng-template>
             </p-toolbar>
-
-            <p-table #dt [value]="users" [columns]="cols" responsiveLayout="scroll" [rows]="10" [globalFilterFields]="['name','country.name','representative.name','status']" [rows]="10" [paginator]="true" [rowsPerPageOptions]="[10,20,30]" [showCurrentPageReport]="true" currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries" [(selection)]="selectedUsers" selectionMode="multiple" [rowHover]="true" dataKey="id">
+            <p-table #myTable [value]="OrganizationsUsersList" [columns]="cols" [loading]='loading'
+                [totalRecords]="OrganizationsUsersList.length" responsiveLayout="scroll" [rows]="10" [paginator]="true"
```

Comment:
@poojakarma Can you remove this page number also [rows]="10" from the HTML file

Reply:
ok sir